plugins/idea/src/main/java/org/robovm/idea/RoboVmPlugin.java

```diff
@@ -223,12 +231,16 @@ public void run() {
         });
     }
 
+    private static String getNowString() {
```

Comment:
I would rename it to something like `getFormattedTimeStamp` just to keep it same as everywhere in the world

Reply:
Fixed.